index.html

```diff
@@ -2021,6 +2021,13 @@ <h3>Securing Mechanisms</h3>
 algorithm</a> section.
           </li>
           <li>
+MUST document post-processing considerations for data returned from verification
+processes. This includes, but is not limited to, what information has been
+secured, for what period of time after verification, when or what additional
+information might be required to assist in decision making, and when
+care is necessary even when the returned data was properly secured.
```

Comment:
Business logic (a/k/a "decision making") is known to be out-of-bounds for this specification, and I think that includes the output of verification processes, so "required" and "necessary" should not be in this text. "Caution" and/or "recommendation", on the other hand, seems fair game.
```suggestion
information might be recommended to assist in decision making, and when
caution is advised even when the returned data was properly secured.
```